app/build.gradle

```diff
@@ -150,8 +151,9 @@ dependencies {
     implementation "com.github.bumptech.glide:glide:$glideVersion"
     kapt "com.github.bumptech.glide:compiler:$glideVersion"
 
-    def nearbyVersion = "17.0.0"
-    implementation "com.google.android.gms:play-services-nearby:$nearbyVersion"
+    //def nearbyVersion = "17.0.0"
+    //implementation "com.google.android.gms:play-services-nearby:$nearbyVersion"
```

Comment:
Please read [Best practices](https://wiki.sinnerschrader.com/display/SWP/Best+practices) and avoid ignoring it 🙂 
#UNUSED
Detele it or add a comment (todo for example) why it's commented and what do you expect in the future with this code.